app/services/solidus_pay_tomorrow/client/create_order_service.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module SolidusPayTomorrow
+  module Client
+    class CreateOrderService < SolidusPayTomorrow::Client::BaseService
+      attr_reader :order
+
+      CREATE_ENDPOINT = 'api/application/ecommerce/orders'
+
+      # @param order [Spree::Order] Spree order
+      # @param payment_method [SolidusPayTomorrow::PaymentMethod]
+      def initialize(order:, payment_method:)
+        @order = order
+        super
+      end
+
+      def call
+        create
+      end
+
+      private
+
+      def create
+        handle_errors(HTTParty.post(uri, headers: auth_headers, body: create_body.to_json))
+      end
+
+      def uri
+        "#{api_base_url}/#{CREATE_ENDPOINT}"
+      end
+
+      def create_body
+        { orderId: order.number,
+          # TODO: We'll need to implement firstname and lastname. They are deprecated in Solidus
+          firstName: order.bill_address.name,
+          lastName: order.bill_address.name,
+          street: order.bill_address.address1,
+          city: order.bill_address.city,
+          zip: order.bill_address.zipcode,
+          state: order.bill_address.state.abbr,
+          email: order.email,
+          # TODO: Update URLs with actual routes once controllers are written
+          returnUrl: 'https://success-redirect-url.com',
+          cancelUrl: 'https://cancelled-order-redirect-url.com',
+          notifyUrl: 'https://webhook-notify-url.com',
```

Comment:
Probably the domain will be every time the same. We can consider to use a constant for path

Reply:
Used in https://github.com/nebulab/solidus_pay_tomorrow/pull/7/commits/390014a30a052261cae08adcd7221d7641aaf79f